src/content/en/2021/caching.md

```diff
@@ -17,4 +20,435 @@ featured_stat_3: TODO
+  sheets_gid="856268091",
+  sql_file="resource_age_party_and_type_wise_groups.sql"
+) }}
+
+The largest group of cached 1st party CSS (32.2%) tended to be 8-52 weeks old, while the largest group for 3rd parties was less than a week with 51.8% of resources cached for that duration.
+
+Finally, HTML has the largest 1st party group served with less than a week with 42.7% and 3rd party's largest group is between 1-7 weeks with 43.1%.
+
+Considerations after reviewing this data: 
+- The freshest content for 1st party is HTML while for 3rd party it is CSS. 
+- The most stale content for both 1st and 3rd party is Images.
+
+This data shows us that 1st parties have prioritized refreshing HTML content, which usually holds the link to JS and CSS files, while 3rd party providers that are mostly CSS and script-driven, like browser extensions, have prioritized keeping their CSS up-to-date. When we consider the origins behind 1st parties vs. 3rd parties, it follows that the _way_ content is delivered may be more important to 3rd parties than the actual content, thus making their presentation and optimization of it, all the more important. 
+
+Mobile resources with a cache TTL that was considered too short compared to its content age have seen an improvement since 2020. This data is exciting because it hints at the community's growing understanding of appropriate caching. While a cache TTL that is too long is unhelpful, it is also less useful if it is too short. The correlation between cache TTL and content age is slowly closing the gap, moving from 60.2% in 2020 to 54% in 2021. The more we can pay attention to content's age, the more accurately we can set cache limits.
+
+{{ figure_markup(
+  caption="TODO",
+  content="54%",
+  classes="big-number",
+  sheets_gid="768623684",
+  sql_file="todo.sql"
+) }}
+
+Developers are getting better at setting the cache duration more accurately to the content age, resulting in more responsible, and therefore more effective, caching.
+
+<figure>
+  <table>
+    <tr>
+     <th>Client</th>
+     <th>1st party</th>
+     <th>3rd party</th>
+     <th>Overall</th>
+    </tr>
+    <tr>
+     <td>Desktop</td>
+     <td class="numeric">59.3%</td>
+     <td class="numeric">46.2%</td>
+     <td class="numeric">54.3%</td>
+    </tr>
+    <tr>
+     <td>Mobile</td>
+     <td class="numeric">60.1%</td>
+     <td class="numeric">44.7%</td>
+     <td class="numeric">54.3%</td>
+    </tr>
+  </table>
+  <figcaption>TODO: Caption the table above</figcaption>
+</figure>
+
+When we split the data between first and third party providers, the largest improvements come from third parties where we have a 13.2% improvement. It is very encouraging to see companies around the world building products for developers that are highly optimized on caching. It's possible that the developer community's increased attention towards improving performance has encouraged and even incentivized third parties to optimize their caching strategies.
```

Comment:
Edit: 

When we split the data between 1st and 3rd party providers, the largest improvements come from 3rd parties where we have a 13.2% improvement. It is highly encouraging to see companies around the world building products for developers that are optimizing caching. It's possible that the developer community's increased attention towards improving performance has encouraged and even incentivized 3rd parties to optimize their caching strategies.

However, the challenge remains for how 1st parties can effectively improve over the coming years.
(Add something here???-------------------)